test/utils/utils.go

```diff
@@ -225,7 +225,7 @@ func (p *PodmanTest) WaitContainerReady(id string, expStr string, timeout int, s
 			return false
 		}
 
-		if strings.Contains(s.OutputToString(), expStr) {
+		if strings.Contains(s.OutputToString(), expStr) || strings.Contains(s.ErrorToString(), expStr) {
```

Comment:
This looks a bit weird, why are we matching stderr for a successful case. Should it be a different commit ?

Reply:
Never mind I think reason for this is specified  in the commit [Update the registry server we test against from 2.6 to 2.8](https://github.com/containers/podman/pull/15108/commits/5a5624f8189c0e4c529e4c00d6ed66397d1db5b4) .

Reply:
Yes, that’s the reason. `WaitContainerReady` is most commonly used for the registry, so I took this fairly lazy route.

Alternatively, we could add an option whether to check `stderr` to this function, or maybe a specialized “start a registry and wait until ready” helper.